business_requirement_deliverable/models/business.py

```diff
@@ -66,6 +66,7 @@ class BusinessRequirementResource(models.Model):
         string='Business Requirement',
         store=True
     )
+    unit_price = fields.Float(string='Cost Price')
```

Comment:
@darshan-serpent Could you add `groups` parameter like we had on BRDCost?

I think should be add this group
https://github.com/OCA/business-requirement/blob/8.0/business_requirement_deliverable/security/business_requirement_deliverable_security.xml#L5

What do you think @elicoidal ?